browser_history/utils.py

```diff
@@ -156,3 +156,33 @@ def default_browser():
     # nothing was found
     logger.warning("Current default browser is not supported")
     return None
+
+
+def get_browser(browser_name):
+    """
+    This method returns the browser class from a browser name.
+
+    :param browser_name: a string representing one of the browsers supported
+        or ``default`` (to fetch the default browser).
+
+    :return: A browser class which is a subclass of
+        :py:class:`browser_history.generic.Browser` or ``None`` of no supported
+        browsers match the browser name given
+
+    :rtype: union[:py:class:`browser_history.generic.Browser`, None]
```

Comment:
This can be added as a type hint too

Reply:
Sure, will add it.

Reply:
Adding hinting causes circular import error

Reply:
Oh okay. I tried [forward referencing](https://www.python.org/dev/peps/pep-0484/#forward-references) and it seems to work (I added the type hint as `"generic.Browser"`).

Reply:
There seems to be a weird pylint issue causing it to fail in python 3.9 if what is being said [here](https://github.com/PyCQA/pylint/issues/3882) is anything to go by

Reply:
The discussion on that thread seems to have gotten a little toxic.

I guess we can leave this type hint for now. We could create a `good-first-issue` or `first-timers-only` issue for this xD.

Reply:
>The discussion on that thread seems to have gotten a little toxic.

They redeemed themselves though, I understand the guy's frustrations but I also see where the pylint guys are coming from. 
>We could create a good-first-issue or first-timers-only issue for this xD.

Yeah, we could create an issue for that. Lemme just remove the hint for now.